rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] Please make sure that you have established the `association` correctly, your `Rental` is an intermediate class for the person and book, so that is why you need to also maintain the track of `rentals` in the `person` and `book` class, because if you do it then it will be easier for your to not only get the details of the person or book with rentals details, but with the help of person or book class, you can get the appropriate the rentals details more specifically.